src/client/hooks/useAllowance.ts

```diff
@@ -0,0 +1,60 @@
+import { useState, useEffect, useContext } from 'react';
+import isEqual from 'lodash/isEqual';
+
+import { AppServiceContext } from '@context';
+import { NetworkSelectors, WalletSelectors } from '@store';
+import { TokenAllowance } from '@types';
+import { useAppSelector } from '@hooks';
+
+import { usePrevious } from './usePrevious';
+
+export function useAllowance(
```

Comment:
Can we have here named params?

Reply:
solved on ffefab04728c259b147ce88165c34d609bd1c7b4